_posts/en/newsletters/2021-11-24-newsletter.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Bitcoin Optech Newsletter #176'
+permalink: /en/newsletters/2021/11/24/
+name: 2021-11-24-newsletter
+slug: 2021-11-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a discussion about how to allow LN users
+to choose between higher fees and higher payment reliability and
+includes our regular sections with popular questions and answers from
+the Bitcoin StackExchange, announcements of new releases and release
+candidates, and summaries of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **LN reliability versus fee parameterization:** Joost Jager started a
+  [thread][jager params] on the Lightning-Dev mailing list about how to
+  best allow users to choose between paying more fees for faster
+  payments or waiting longer to save money.  One of the challenges
+  discussed in the thread is how to relate user preferences that exist
+  on a single continuum to the discrete and multi-factor routes being
+  returned by a pathfinding algorithm.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why is it important that nonces when signing not be related?]({{bse}}110811)
+  Pieter Wuille provides mathematical breakdowns for how private key information
+  can be leaked when signing twice with the same public key using: the same nonces,
+  nonces with known offsets, and nonces with known factors.
```

Comment:
I think the final section (**Arbitrary known relation between the nonces**) is also interesting, particularly the conclusion: "But in between those two, there is a huge gap of techniques which are neither known to be broken, nor proven secure. Many of them might well be secure, but we don't know." Perhaps something summarising that could be included?

Reply:
@jnewbery I pushed an addition for this, but welcome your feedback